lib/routes/account.js

```diff
@@ -406,177 +398,48 @@ module.exports = (log, db, mailer, Password, config, customs, checkPassword, pus
         const form = request.payload
         const email = form.email
         const authPW = form.authPW
-        const service = request.payload.service || request.query.service
-        const redirectTo = request.payload.redirectTo
-        const resume = request.payload.resume
-        const ip = request.app.clientAddress
-        const requestNow = Date.now()
         const originalLoginEmail = request.payload.originalLoginEmail
         const verificationMethod = request.payload.verificationMethod || request.query.verificationMethod
+        const requestNow = Date.now()
 
-        let needsVerificationId = true
-        let accountRecord, sessions, sessionToken, keyFetchToken, mustVerifySession, doSigninConfirmation,
-          unblockCode, customsErr, didSigninUnblock, tokenVerificationId, tokenVerificationCode
-
+        let accountRecord, password, sessionToken, keyFetchToken, didSigninUnblock
```

Comment:
I'm extremely pleased to have removed many of these local variables, pushing them down into more focused functions inside the new helper leave.